### ISSUE TITLE
Support using images

### ISSUE THREAD

**Issue by rclai** [jeanregisser/react-native-slider#12]
On iOS, you can set a thumb image, and you can set the left side and right side image, where you would for example create a slider image that includes the left and right caps, then in the middle, you would have two 1 pixel wide by __ pixel high pictures that would tile the left and right side of the thumb as you move. Don't know how feasible this would be?

**Comment by amibition521** [comment]
I agree your advise,I want to achieve such effect.But I met problems, both out of step.
![20151227114734](https://cloud.githubusercontent.com/assets/13275317/12009563/376178ca-acb8-11e5-90bb-e02803132480.png)
What are your thoughts?

**Comment by lewisbarber** [comment]
+1
